src/pawns.cpp

```diff
@@ -45,18 +40,24 @@ namespace {
 
   // Backward pawn penalty by opposed flag
   const Score Backward[2] = { S(67, 42), S(49, 24) };
+  
+  // Unsupported pawn penalty, for pawns which are neither isolated or backward.
```

Comment:
Instead here we don't need the dot :-)  On one liners no dot, on multi-line then add dot